Day-01/go/thezexquex/one.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"slices"
+	"strconv"
+	"strings"
+)
+
+func main() {
+	content, _ := os.ReadFile("Day-01/input.txt")
```

Comment:
Please place the input file in the same folder as the code

```suggestion
	content, _ := os.ReadFile("input.txt")
```

Reply:
Changed it to `Day-01/go/thezexquex/input.txt`